src-graphql/base64.js

```diff
@@ -0,0 +1,7 @@
+export default function base64(str) {
+  if (typeof btoa === 'function') {
+    return btoa(str);
+  } else {
+    return Buffer.from(str).toString('base64');
```

Comment:
I believe since we shim in `btoa` you can just safely use `btoa` in Node land
https://github.com/Shopify/js-buy-sdk/blob/f958d6b79873b5770413ceceef0fe84acd525794/src/polyfills.js